spec/date_version_calculator_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'spec_helper'
+
+describe Fastlane::Wpmreleasetoolkit::Versioning::DateVersionCalculator do
+  describe 'calculates the next release version when using date versioning' do
+    context 'when the current month is not December' do
+      it 'increments the minor version number without prompting the user' do
+        allow(Time).to receive(:now).and_return(Time.new(2024, 4, 15))
+        version = Fastlane::Models::AppVersion.new(2005, 13, 1, 1)
+        bumped_version = described_class.new.next_release_version(after: version)
+        expect(bumped_version.to_s).to eq('2005.14.0.0')
+      end
```

Comment:
It's a bit confusing to have the current mocked year (2024) and the major version (2005) be so different 😅 

I guess that's intentional (?), because even if we're in early 2024 maybe we're still technically working on a 2023.x version whose release got postponed or whatnot, or today is 2024.01.02 but we're still releasing 2023.14.0`…) so there's no guarantee that the years will exactly match?
But still feels odd, especially with such a diff between 2024-2005 = 19 years 😄 

I think it should be either just `24.14.0.0` (dropping the century; and maybe in 2123 we'll just use `124`), or match the

Reply:
Ah sorry, the 2005 was really just a random year because it should work either way. I see your point for readability, so I updated the test dates to match up.

Reply:
Addressed in https://github.com/wordpress-mobile/release-toolkit/commit/f18258feed621c09161ee2c9dc180ba8d1517199